incomplete-cat-B-items.md

```diff
@@ -0,0 +1,116 @@
+|  B  |          | 6.6.8     | active window button textures   | `window.active.button.toggled.bg`                       |
+|  B  |          | 6.7.1     | inactive window button textures | `window.inactive.button.unpressed.bg`                   |
+|  B  |          | 6.7.2     | inactive window button textures | `window.inactive.button.pressed.bg`                     |
+|  B  |          | 6.7.3     | inactive window button textures | `window.inactive.button.hover.bg`                       |
+|  B  |          | 6.7.4     | inactive window button textures | `window.inactive.button.disabled.bg`                    |
+|  B  |          | 6.7.5     | inactive window button textures | `window.inactive.button.toggled.unpressed.bg`           |
+|  B  |          | 6.7.6     | inactive window button textures | `window.inactive.button.toggled.pressed.bg`             |
+|  B  |          | 6.7.7     | inactive window button textures | `window.inactive.button.toggled.hover.bg`               |
+|  B  |          | 6.7.8     | inactive window button textures | `window.inactive.button.toggled.bg`                     |
+|  B  |          | 6.8.1     | menu colors                     | `menu.title.text.color`                                 |
+|  B  |          | 6.8.3     | menu colors                     | `menu.items.disabled.text.color`                        |
+|  B  |          | 6.8.5     | menu colors                     | `menu.items.active.disabled.text.color`                 |
+|  B  |          | 6.9.3     | menu textures                   | `menu.title.bg`                                         |
+|  B  |          | 6.10.3    | osd textures                    | `osd.hilight.bg`                                        | for desktop osd only
+|  B  |          | 6.10.4    | osd textures                    | `osd.unhilight.bg`                                      | for desktop osd only
+|  B  |          | 6.11.2    | osd colors                      | `osd.hilight.bg.color`                                  | for desktop osd only
+|  B  |          | 6.11.3    | osd colors                      | `osd.unhilight.bg.color`                                | for desktop osd only
+|  B  |          | 6.12.2    | text justification              | `menu.title.text.justify`                               |
+|  B  |          | 7.1.2     | menu                            | `label`                                                 |
+|  B  |          | 7.1.4     | menu                            | `execute`                                               |
+|  B  |          | 7.3.2     | menu separator                  | `<separator label="" />`                                | For separator with text
+|  B  |          | 8.1.2     | extra                           | Support png buttons                                     |
+|  B  |          | 8.2.1     | extra                           | Read menu.xbm if it exists                              |
```

Comment:
I think we are reading `menu.xbm` already, not 100% certain right now though.

Reply:
Just checked. Yes we do. Should be a "complete"